src/shipit_pulse_listener/shipit_pulse_listener/__init__.py

```diff
@@ -1,5 +1,6 @@
 from cli_common.log import init_logger
+import logbook
 
 
 # Setup logger for bot execution
-init_logger()
+init_logger(level=logbook.INFO)
```

Comment:
I think it would make sense to turn level to ``logbook.INFO`` and in please command we set the level to ``logbook.WARNING``. what do you think?